doc/manpages/man1/qrest.1

```diff
@@ -11,7 +11,7 @@ There are 2 ways to entered the tempo value :
 .IP \(bu 2
 Keyboard numerical input : User simply types tempo value in a text field and hit \'enter\'.
 .IP \(bu 2
-Realtime button click : User clicks a button in sync with beats, and qrest will tell her if she taps steadily.
+Realtime button click : User clicks a button in sync with beats, and qrest will tell her/him if she/he taps steadily.
```

Comment:
using she/her is the usual 'polite' way of refering to anyone, male or female. I won't merge this change